File: python_programming/notes/Conditionals_notes.py
# LC 7th Conditional notes

# What puts something inside of the "if" statement?
# if your are asking a question and want a specific statement for a specific answer
# A.K.A. Tab

# What do "if" statements do?
# Tells code to check for true of false
# Example
#homework_done = input("is your homework done: ").strip().capitalize()

#if homework_done == "Yes":
    #print("Yes you can go!")


# What are boolean statements?
# They always show as a true or false

# if not the answer for if statement then does this instead
# Example
# What do else statements do
#else:
    #print("Then go do your homework")

# What kind of statement do you use if you have more than 2 needed outcomes?
# Then do your least likly outcome first 

# What do each of the differents symbols mean in Conditionals?
# Example 
grade = 110

if grade >= 90:
    if grade > 100:
        print ("You cheated that isn't possible!")
    else:
        print(f"You have {grade}% that is an A!")
elif grade >= 80:
    print(f"You have {grade}% that is an B!")
elif grade >= 70:
    print(f"you have {grade}% that is a C!")
else:
    print(f"You have {grade}% that is a D or lower :(")

# What are the 3 logical operatiors for?
# and  
# not  
# or  
# Example
name = input("What is your name: ")

if name == "Ms LaRose":
    print("You are the teacher!")
elif name == "Tia" or name == "Ashley":
    print("You are the TA")
else:
    print(f"Hello {name}, you are a student!")
    if grade >= 70:
        print("And you are passing the class!")

# What are logical operatiors for? 
# so you can meet all the oporations that you need

# What does a nested conditional statement do?
# a condition that only happens inside anothor one

File: c_programming/notes/Conditionals_Notes.c
// LC 7th Conditional Notes
#include <stdio.h>
#include <string.h>

int main(void){
    int grade;
    int name[50];
    printf("What is you grade percent: ");
    scanf("%d", &grade);

    printf("What is you name: ");
    scanf("%s", &name);

    //printf("%d\n", strcmp(name, "Ms.LaRose"))
    if(strcmp(name, "Ms.LaRose") == 0){
        printf("You don't get a grade!\n");
    }else if(grade >= 90){
        printf("You have an A\n");
    }else if(grade >= 80){
        printf("You have an B\n");
    }else if(grade >= 70){
        printf("You have an C\n");
    }else{
        printf("You are failing :(\n");
    }
    return 0;
}


//What puts something inside of the “if” statement?
// The culrly prackets like this {}

//How are conditions written in C?
// they are three ways
// if
// else if
// else

//How do we write elif conditions in C?
// else if

//When do else conditions run?
// if everything before them is false

//What are the 3 logical operators in C?
//and / &&
// or / ||
// not / !